crates/ruff_db/src/diagnostic/mod.rs

```diff
@@ -1312,6 +1312,10 @@ pub struct DisplayDiagnosticConfig {
     show_fix_diff: bool,
     /// The lowest applicability that should be shown when reporting diagnostics.
     fix_applicability: Applicability,
+    /// The program for which diagnostics are being rendered (Ruff or ty).
+    ///
+    /// This is currently only used by the GitHub output format and defaults to `Program::Ty`.
```

Comment:
Refer to my other comment, I'd probably remove the default value for this field.